### PR TITLE
Unity 2019.3.7f1 upgrade

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description

A follow on from #1272, this PR updates the GDK to 2019.3. Again, most of these changes are asset re-imports, sorry 'bout it.  

#### Tests

- [x] Release QA pipeline
- [x] iOS client
- [x] MacOS client 
- [x] Windows client 

#### Documentation
 
- [x] Changelog
- [x] Upgrade guide
- [x] Docs update

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+- The GDK for Unity now supports Unity 2019.3. You will need to upgrade your Unity project. [#1272](https://github.com/spatialos/gdk-for-unity/pull/1272)
```
Comment (paulbalaji) [inline CHANGELOG.md]:
```suggestion
- The minimum version of Unity supported by the GDK is now 2019.3. You will need to upgrade your Unity project. [#1272](https://github.com/spatialos/gdk-for-unity/pull/1272)
```

Comment (paulbalaji) [reply CHANGELOG.md]: makes it clearer why they have to upgrade?

Comment (jamiebrynes7) [reply CHANGELOG.md]: Ye

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+- The minimum version of Unity supported by the GDK is now 2019.3. You will need to upgrade your Unity project. [#1272](https://github.com/spatialos/gdk-for-unity/pull/1272)
```
Comment (jamiebrynes7) [inline CHANGELOG.md]:
```suggestion
- The minimum version of Unity supported by the GDK is now 2019.3. You will need to upgrade your Unity project. [#1336](https://github.com/spatialos/gdk-for-unity/pull/1336)
```

Comment (paulbalaji) [reply CHANGELOG.md]: haha

**Comment by jamiebrynes7** [comment]
👋 @MarkWoulfeImprobable, @anne-edwards, @ElleEss - I've added you to this PR since we are going to have tech writer approval on changelogs from now on. 

You can find the changelog change here https://github.com/spatialos/gdk-for-unity/pull/1336/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR7 :)

**Comment by jamiebrynes7** [comment]
Also @SPPWilliams ☝️ now that you have access!

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+- The minimum version of Unity supported by the GDK is now 2019.3. You will need to upgrade your Unity project. [#1336](https://github.com/spatialos/gdk-for-unity/pull/1336)
```
Comment (anne-edwards) [inline CHANGELOG.md]:
```suggestion
- The minimum version of Unity that the GDK supports is now 2019.3. You need to upgrade your Unity project. [#1336](https://github.com/spatialos/gdk-for-unity/pull/1336)
```

File: UPGRADE_GUIDE.md [hunk 4]
```diff
@@ -1,5 +1,11 @@
 # Upgrade Guide
 
+## From `0.3.4` to `0.3.5`
+
+### Unity 2019.3 upgrade
+
+You must upgrade your project to 2019.3 to be able to use this, and future versions, of the GDK for Unity. Any patch release of 2019.3 should work, but we test against 2019.3.7f1.
```
Comment (anne-edwards) [inline UPGRADE_GUIDE.md]:
```suggestion
You must upgrade your project to 2019.3 to be able to use this version, and future versions, of the GDK for Unity. Any patch release of 2019.3 should work, but we test against 2019.3.7f1.
```

**Pull request merged**